ui/src/app/services/notebooks-swagger-fetch-clients.ts

```diff
@@ -10,7 +10,7 @@ import {
   BaseAPI,
   ClusterApi,
   Configuration as FetchConfiguration,
-  FetchAPI
+  FetchAPI, JupyterApi, NotebooksApi
```

Comment:
nit: I'd keep these one per line. The way it is now it looks like they are set apart from the rest of the list somehow.

Reply:
yep, accident